robyn/argument_parser.py

```diff
@@ -87,6 +88,8 @@ def __init__(self) -> None:
             if arg.endswith(".py"):
```

Comment:
@VishnuSanal , this is what stops the app from running in module mode

e.g.

`robyn -m src --dev` will not work if all the code is present in the `src/__main__.py` file

Reply:
> this is what stops the app from running in module mode

yep, the missing parameter is `file_path`. this is why I have assigneed it in the else block & did worksrounds to make it work on other parts of the code.